unitary/examples/tictactoe/tic_tac_toe.py

```diff
@@ -227,9 +229,110 @@ def sample(self, count: int = 1) -> List[str]:
         """
         return [_result_to_str(result) for result in self.board.peek(count=count)]
 
-    def print(self) -> str:
+
+class GameInterface:
+    """
+    A class that provides a command-line interface to play Quantum Tic Tac Toe.
+
+    Initialize by providing an instance of a TicTacToe game, then call play()
+    to run the game.
+
+    Args:
+        game: A TicTacToe instance for the game interface to wrap.
+        file:  Optional IOBase file object to write output to.
+            This enables the battle to write status to a file or string
+            for testing.
+
+    """
+
+    help_text = """
+    You can enter:
+    - 1 character from [abcdefghi] to place a mark in the corresponding square (eg "a")
+    - 2 characters from [abcdefghi] to place a split mark in corresponding squares (eg "bd")
+    - "map": show board map
+    - "exit" to quit
+    """
+
+    board_map = """
```

Comment:
Typically global constants are capitalized, such as BOARD_MAP (or _BOARD_MAP if it should be package private).  Also, I would put this above GameInterface, as I think it is now a member attribute when defined this way.

See https://google.github.io/styleguide/pyguide.html#316-naming